endpoints/proposal.yaml

```diff
@@ -13,8 +13,6 @@ info:
   version: 0.0.1
   title: Proposal
   description: |
-    * Allows the buyer to submit a Request for Proposal (RFP) to the seller with overall media budget, goal 
-    and makegood terms and optionally include specific detailed inventory requests 
```

Comment:
The proposal endpoint file had an outdated version of the RFP endpoint within it, so I removed it in lieu of making RFPs have their own endpoint and schema files.